client/src/screens/new-group.screen.js

```diff
@@ -181,16 +181,16 @@ const createGroupMutation = graphql(CREATE_GROUP_MUTATION, {
           // Read the data from our cache for this query.
           const data = store.readQuery({
             query: CURRENT_USER_QUERY,
-            variables: { id: ownProps.auth.id },
+            variables: { username: ownProps.auth.username },
```

Comment:
ooh actually we can remove this entirely - we don't pass any variables to `current-user.query`.

Reply:
Aah right. my bad.